client-akka-http/src/main/scala/bekindrewind/akkahttp/AkkaHttpResponseStore.scala

```diff
@@ -0,0 +1,13 @@
+package bekindrewind.akkahttp
+
+import io.circe.{ Decoder, Encoder }
+
+private[akkahttp] case class AkkaHttpResponseStore(httpProtocol: String, contentType: String, data: String)
```

Comment:
I didn't think about these other fields when initially creating the data model. Maybe we should make them standard fields in the VCR record format rather than only for akka-http. Because right now the body for the akka-http client is different than the others and looks like:

```json
"body" : "{\"httpProtocol\":\"HTTP/1.1\",\"contentType\":\"text/html; charset=UTF-8\",\"data\":\"..."
```

which is a little awkward. I mean, worst case I'd want that to be:

```json
"body": {
	"httpProtocol": "HTTP/1.1",
	"contentType": "text/html; charset=UTF-8",
	"data": "..."
}
```

but I think it would be better to move `httpProtocol` in the `VcrRecord` case class, and `contentType` in `VcrRecordResponse`.

Unless you have other suggestions?

Reply:
I was curious and looked at the ruby VCR format. They put the `http_version` in the response object as a top-level field. I suggested `VcrRecord` because the protocol applies to both request and response.

https://relishapp.com/vcr/vcr/docs/cassettes/cassette-format

Reply:
I agree to move `httpProtocol` and `contentType`, so VCR records can be loaded on any implementation.